x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/view/index.tsx

```diff
@@ -126,6 +131,36 @@ export const EndpointList = () => {
   const dispatch = useDispatch<(a: EndpointAction) => void>();
   // cap ability to page at 10k records. (max_result_window)
   const maxPageCount = totalItemCount > MAX_PAGINATED_ITEM ? MAX_PAGINATED_ITEM : totalItemCount;
+  const [showTransformFailedCallout, setShowTransformFailedCallout] = useState(false);
+
+  const getTransformStats = useCallback(async () => {
+    if (!services?.http) {
+      return;
+    }
+
+    const transformStats: TransformStatsResponse = await services.http.get(TRANSFORM_STATS_URL);
```

Comment:
To narrow down what you're requesting, you don't have to fetch all transforms.

You can do `/_transform/endpoint.metadata_current-*/_stats`. The stats API supports `_transform/<transform_id>/_stats` where the `<transform_id>` is allowed to be a partial match with a wildcard.

Reply:
good call, thanks for the tip!

Reply:
ah right, so I totally forgot I glanced at this yesterday 😅. so I'm using https://github.com/elastic/kibana/blob/faed2f6fe10d74273f9ff3a023fabd1c9fee849d/x-pack/plugins/transform/server/routes/api/transforms.ts#L135 and it doesn't support passing the transformId. we also have https://github.com/elastic/kibana/blob/faed2f6fe10d74273f9ff3a023fabd1c9fee849d/x-pack/plugins/transform/server/routes/api/transforms.ts#L164 but that one only accepts a single transformId. thought about adding a new api for supporting the wild card but felt a bit overkill for what we need right now. open to being convinced otherwise.

Reply:
and nm....I'm derping🤪. https://github.com/elastic/kibana/blob/faed2f6fe10d74273f9ff3a023fabd1c9fee849d/x-pack/plugins/transform/server/routes/api/transforms.ts#L164 allows wildcards. will make the update.

Reply:
What happens if you pass in a transformID with a wildcard in the string? Really we should only be getting one result back anyway. Does it break?

Reply:
works as expected.

![Screen Shot 2021-07-30 at 11 13 48 AM](https://user-images.githubusercontent.com/11009772/127681663-281cab73-b033-4529-876f-b73781ab5e51.png)

Reply:
One note. The "`default`" in the name refers to the **namespace** property used when adding the Endpoint integration in fleet. The source data streams can be namespaced, and then people can have segmented source data that way. We have chosen to make the transform namespace-agnostic, by grabbing from all source data streams (so bumping the wildcard farther to the left, so it wildcards both namespace and version/dates).

The _transform_ does this to grab from any _source_ data stream. So the transform itself is pretty much always "default" namespace. 

But that's something to be aware of. Make sure that using a namespaced integration doesn't change the transform's ID to not be "default" anymore.